temporian/implementation/numpy/operators/test/tick_calendar_test.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime, timedelta
+from absl.testing import absltest
+
+from temporian.core.operators.tick_calendar import TickCalendar
+from temporian.implementation.numpy.data.io import event_set
+from temporian.implementation.numpy.operators.tick_calendar import (
+    TickCalendarNumpyImplementation,
+)
+from temporian.implementation.numpy.operators.test.test_util import (
+    assertEqualEventSet,
+    testOperatorAndImp,
+)
+
+
+class TickCalendarOperatorTest(absltest.TestCase):
```

Comment:
It would also be interesting to test invalid situation e.g. seconds=-1, seconds="hello", seconds=60 :)